src/docs/languages.md

```diff
@@ -1,67 +1,19 @@
 ---
 eleventyNavigation:
-  parent: Eleventy Documentation
+  parent: Eleventy Projects
   key: Template Languages
-  order: 7
+  order: 3
 ---
-
 # Template Languages
 
-{% templatelangs templatetypes, page %}
-
-
-## Overriding the Template Language {% addedin "0.2.14" %}
-
-There are a couple of different ways you can tell Eleventy how you want to process a file:
-
-1. The file extension (importantly, this is also used to find files to process).
-2. [Configuration options](/docs/config/):
-   - `markdownTemplateEngine`: The default global template engine to pre-process markdown files. Use `false` to avoid pre-processing and only transform markdown.
-   - `htmlTemplateEngine`: The default global template engine to pre-process HTML files. Use `false` to avoid pre-processing and passthrough copy the content (HTML is not transformed, so technically this could be any plaintext).
-3. `templateEngineOverride` in the template’s front matter. Should be _one_ templating engine (`liquid`) or markdown paired with another templating engine (`liquid,md`). See examples below.
-
-### `templateEngineOverride` Examples
-
-#### Replace with a single templating engine
-
-If your file is called `example.liquid`—instead of `liquid`, this will be parsed as a `njk` Nunjucks template:
-
-{% codetitle "example.liquid" %}
-
-```markdown
----
-templateEngineOverride: njk
----
-```
+Eleventy’s super power is that it is built on an extensible architecture that can work with one or more template syntaxes in the same project.
 
-#### Special case: pairing a templating engine with `md` Markdown
-
-Remember that—by default—Markdown files are processed with an additional preprocessor template engine set globally with the `markdownTemplateEngine` configuration option. So, when using `templateEngineOverride` on markdown files be sure to list each templating engine you’d like to use.
-
-For example, you may want to process `njk` Nunjucks first and then `md` markdown afterwards. Markdown is supported either by itself or with another engine. No other templating engines can be combined in this way—Markdown is the exception here. Any other combination attempt will throw an error.
-
-##### Markdown and nothing else
-
-```markdown
----
-templateEngineOverride: md
----
-```
-
-##### Nunjucks and then Markdown
+{% templatelangs templatetypes, page %}
 
-```markdown
----
-templateEngineOverride: njk,md
----
-```
+_Some of these template types may require plugin installation._
```

Comment:
While true I feel like this could cause a reader to wonder, which languages would be affected.

Reply:
Yeah, I had this same thought but forgot to circle back. Let me figure out something better here.